drivers/i2c/i2c_ll_stm32.c

```diff
@@ -387,8 +387,6 @@ static int i2c_stm32_init(const struct device *dev)
 	}
 
 #ifdef CONFIG_PM_DEVICE_RUNTIME
-	i2c_stm32_suspend(dev);
-	pm_device_init_suspended(dev);
```

Comment:
If the `i2c_stm32_runtime_configure` would retain the `pm_device_runtime_get/put` logic it has currently, it would make more sense to remove only `i2c_stm32_suspend` and leave `pm_device_init_suspended`